JoyboyCommunity/assets/svgs/svgComponents/Info.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import Svg, {SvgProps, Path} from 'react-native-svg';
+const InfoIcon = (props: SvgProps) => (
+  <Svg width={18} height={18} fill="none" {...props} viewBox="0 0 18 18">
+    <Path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
Svg attributes should be camelCased, for example `fill-rule` should be `fillRule`